coalib/results/Diff.py

```diff
@@ -375,12 +375,25 @@ def add_lines(self, line_nr_before, lines):
 
         linediff = self._get_change(line_nr_before, min_line=0)
         if linediff.add_after is not False:
-            raise ConflictError('Cannot add lines after the given line since '
-                                'there are already lines.')
+            raise ConflictError('Cannot add line(s) after the given line since '
+                                'there are already line(s).')
```

Comment:
@aptrishu told to look for single line ConflictError as well.
I think we should go with `line(s)` . This will make things generic.
Otherwise it would require another `condition check` to see if the array contains only one element and to assert different `ConflictError` message for that which doesn't makes sense.